crowbar_framework/app/assets/stylesheets/branding.scss

```diff
@@ -1,2 +1,15 @@
 @import "bootstrap/variables";
 @import "bootstrap/mixins";
+#nodegroups {
+  .panel>.list-group .list-group-item, .panel>.panel-collapse>.list-group .list-group-item {
```

Comment:
Selector should have depth of applicability no greater than 2, but was 5<br>Each selector in a comma sequence should be on its own single line